db/fixtures/showback_usage_types.yml

```diff
@@ -4,6 +4,10 @@
   :measure: "CPU"
   :dimensions: ["average", "number", "max_number_of_cpu"]
 - :category: "Vm"
+  :description: "Template Showback of VM"
+  :measure: "TEMPLATE"
+  :dimensions: ["number","memory_reserve"]
```

Comment:
You can use any unit, if it is not identified we won't be able to translate it to K,M, etc